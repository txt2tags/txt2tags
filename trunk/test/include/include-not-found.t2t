
%!include: UFO.t2t
I will never get here.
