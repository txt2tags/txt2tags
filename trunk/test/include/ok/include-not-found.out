txt2tags: Error: Cannot read file: UFO.t2t
